libraries/lib-wave-track/CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ set( SOURCES
    WaveTrack.h
    WaveTrackSink.cpp
    WaveTrackSink.h
+   WaveTrackUtilities.cpp
+   WaveTrackUtilities.h
```

Comment:
I think smaller is often better for classes.

The function I move into that file was defined in an earlier commit without intrusion into WaveTrack.  It wasn't necessary.  The existing public interfaces were sufficient to define it.

So then, why disturb and complicate the interface of WaveTrack without need?  I'm making the function available for reuse but keeping it separate from the class.

I think we may identify more inessential member functions of WaveTrack that could also be moved out.  It's a big file doing too much.

I think it would be better to have a smaller WaveTrack class someone could study, which provides just the essential services and maintains important class invariants.

Reply:
No strong objection to that.